ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/privacy/PrivateTransactionHandler.java

```diff
@@ -59,15 +58,15 @@ public PrivateTransactionHandler(
 
   public Transaction handle(
       final PrivateTransaction privateTransaction, final Supplier<Long> nonceSupplier)
-      throws IOException {
+      throws Exception {
     LOG.trace("Handling private transaction {}", privateTransaction.toString());
     final SendRequest sendRequest = createSendRequest(privateTransaction);
     final SendResponse sendResponse;
     try {
       LOG.trace("Storing private transaction in enclave");
       sendResponse = enclave.send(sendRequest);
-    } catch (IOException e) {
-      LOG.error("Failed to store private transaction in enclave", e);
+    } catch (Exception e) {
+      LOG.error("Failed to store private transaction in enclave {}", e.getMessage());
       throw e;
```

Comment:
Should this throw a new exception that wraps e and includes the log message?

Reply:
Potentially this can be done, but that is for another PR. The exception is caught in `EeaSendRawTransaction` and converted to a JSON-RPC response using `e.getMessage()`